### PR TITLE
Added documentation

### PR DESCRIPTION
**Pull request by murphybytes**
Connects to heroku/opex-team#363

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,54 @@
...
+  -prom-url string
+        Prometheus URL
+  -statsd-addr string
+        UDP port for statsd listener
+  -version
+        print version string
+```
+Language metrics are supported for Go, JVM, and Ruby. 
+
+#### Go Metrics
```
Comment (apg) [inline README.md]: It really doesn't make sense to document go metrics, or other language metrics here. It acts as an intermediary.

Comment (apg) [reply README.md]: Therefore, should probably not be in here.

File: README.md [hunk 2]
```diff
@@ -4,7 +4,54 @@
 
 ## Overview
 
-*To be written*
+Agentmon reads language metrics from Prometheus or a StatsD source and posts them to the language metrics sink, for
```
Comment (jmorrell) [inline README.md]: Questions I have:

At a high level why is agentmon necessary? My understanding is that for Node agentmon will be used to consolidate metrics from multiple processes. From the description I assume that it allows you to re-use existing Prometheus or StatsD metrics libraries that report in that format?

How do I get Agentmon onto a dyno? Should I add it to a Node buildpack?

How can I get this running locally and run tests if I want to make a change?

[the remark this answers is not in the document]
Comment (murphybytes) [reply README.md]: I think running Prometheus or a statsd server is outside the scope of this document.  I've provided links to relevant documentation instead.

**Pull request merged**